website/functions/get-discourse-comments.js

```diff
@@ -3,6 +3,7 @@ const axios = require('axios')
 const { DISCOURSE_DEVBLOG_API_KEY , DISCOURSE_USER_SYSTEM } = process.env
 const DEVBLOG_URL = 'https://docs.getdbt.com/blog/'
 const DISCOURSE_TOPIC_ID = 2
+const DISCOURSE_EXTERNAL_ID_SUFFIX = '-testing3'
```

Comment:
Are we thinking that this suffix would be blank in prod? If you let me know which slugs are already burned from testing I can try and hard delete them or get the support team to hard delete them

Reply:
Ideally yes, in production we wouldn't have to use a suffix. Let me see if I can pull a list of the external_ids that are currently set in Discourse.